### DISCUSSION TITLE
Chapter 12. 데이터베이스 응용 기술

### DISCUSSION THREAD

**Discussion by crtEvent** [CodeSquad-2023-BE-Study/DB-Study#16] [완료]
Ask a question

**Comment by crtEvent** [comment]
# 04 멀티미디어 데이터베이스 시스템
유튜브는 그 많은 영상 데이터들을 어떻게 저장하고 어떻게 빨리 불러올 수 있을까?

**Reply by crtEvent** [reply]
Google File System


파일을 작은 단위(Chunk)로 나누어 여러 DB에 복제하여 저장
불러온 땐 각 청크들을 동시에 불러와서 빠르게 불러오는 듯
[한글 자료 간단 요약본](https://ryufree.tistory.com/205)
[논문?](https://static.googleusercontent.com/media/research.google.com/ko//archive/gfs-sosp2003.pdf)
[[논문 리뷰] The Google File System(GFS) (1)](https://ingu627.github.io/paper/hadoop/gfs1/)
[[논문 리뷰] The Google File System(GFS) (2)](https://ingu627.github.io/paper/hadoop/gfs2/)

**Reply by crtEvent** [reply]
### 관심있는 주제
하둡
SQL Injection
Google File System
암호화 방식

**Comment by leegyeongwhan** [comment]
데이터 웨어 하우스에 대해 관심이 가는데 aws나 google에서는 클라우드 서비스로 지원을 해주네요.
요즘같은 빅데이터  대용량 데이터를 처리하는 시대에서 데이터 웨어하우스를 둬 (클라우드 서비스를 이용하면 쉬울지는 모르겠지만 좋을것 같습니다) 데이터의 집합을 관리하기 쉬울 것 같습니다. 비지니스의 방향성을 정하기도 좋을 것 같네요.
